dev-app-post/main/default/assignmentRules/Case.assignmentRules-meta.xml

```diff
@@ -54,5 +54,102 @@
             </criteriaItems>
             <template>unfiled$public/SUPPORTNewassignmentnotificationSAMPLE</template>
         </ruleEntry>
+        <ruleEntry>
+            <assignedTo>Regular</assignedTo>
+            <assignedToType>Queue</assignedToType>
+            <criteriaItems>
+                <field>Case.RecordTypeId</field>
```

Comment:
Shouldnt it be recordType.Name?

Reply:
@yashrohan-cgi  The ticket is to pull the changes in the branch from Prod. No changes are done in the rule.